packages/amplify-prompts/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "2.6.5",
   "description": "Utility functions for Amplify CLI terminal I/O",
   "main": "lib/index.js",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
this is no-op for this package, but it will help us not forget about this when we remove the "alias package"